src/ui/Button/Button.tsx

```diff
@@ -1,6 +1,6 @@
-import style from "./button.module.scss"
 import { Link } from 'react-router-dom';
 import { ReactNode, MouseEvent } from 'react';
+import style from "./button.module.scss"
```

Comment:
```suggestion
import style from "./styles.module.scss"
```

По всему проекту исправь

Reply:
исправленно